### PR TITLE
adjustments to caplin peer count

### PR DESCRIPTION
**Pull request by domiwei**
need more peers to publish gossip msgs

### CODE REVIEW

File: cl/sentinel/sentinel.go [hunk 1]
```diff
@@ -242,14 +243,18 @@ func New(
 		return nil, err
 	}
 
+	maxSubnetCount := max(
```
Comment (Giulio2002) [inline cl/sentinel/sentinel.go:246]: I think this should be `cfg.NetworkConfig.AttestationSubnetCount` + `cfg.BeaconConfig.SyncCommitteeSubnetCount` + `cfg.BeaconConfig.MaxBlobsPerBlock`

**Comment by Giulio2002** [comment]
overall good but left just one comment

**Comment by Giulio2002** [comment]
maybe - do `*4` too

File: cl/sentinel/sentinel.go (hunk 1 above)
Comment (domiwei) [reply cl/sentinel/sentinel.go]: hmm, make sense

**Comment by domiwei** [comment]
> maybe - do `*4` too

sure. done

**Review by Giulio2002 — APPROVED** [review]

**Pull request merged**